packet/models.py

```diff
@@ -49,6 +49,13 @@ def by_username(cls, username: str):
         """
         return cls.query.filter_by(rit_username=username).first()
 
+    @classmethod
+    def get_all(cls):
+        """
+        Helper method to get all freshmen easily
+        """
+        return cls.query.all()
```

Comment:
Not sure if the couple of characters saved by this helper method is worth the extra level of abstraction/indirection. Not a big deal either way though.